scripts/Imaging/imgpy/filters/circular_mask.py

```diff
@@ -0,0 +1,106 @@
+from __future__ import (absolute_import, division, print_function)
+import numpy as np
+import helper as h
```

Comment:
I will consider this, although I don't think it would be an issue, from a brief search for whole word + matching case it gives back only usages of helper:
![image](https://cloud.githubusercontent.com/assets/9135965/24542622/3bb03a46-15f4-11e7-9cdc-b14a1c045c3c.png)